app/controllers/api/v1/merchants_controller.rb

```diff
@@ -4,10 +4,34 @@ def index
         serialized_merchants = MerchantSerializer.new(merchants)
         render json: serialized_merchants
     end 
-
+    
     def show
-        items = Item.where(merchant_id: params[:id])
-        serialized_items = ItemSerializer.new(items)
-        render json: serialized_items
+        merchant = Merchant.find(params[:merch_id])
+        serialized_merchant = MerchantSerializer.new(merchant)
+        render json: serialized_merchant
+    end 
+
+    def find
+        merchant = Merchant.find_by(merch_params)
+        serialized_merchant = MerchantSerializer.new(merchant)
+        render json: serialized_merchant
+    end 
+
+    def find_all
+        merchants = Merchant.where(merch_params)
+        serialized_merchants = MerchantSerializer.new(merchants)
+        render json: serialized_merchants
+    end 
```

Comment:
This method feels very similar to `index` with a certain amount of scope. You may be able to roll it into `index` in order to keep your controller ReSTful.